coopr-docs/docs/source/release-notes/release-0.9.9.rst

```diff
@@ -25,14 +25,135 @@
 Welcome to the Coopr 0.9.9 release. Release notes with links to issue details can be seen
 from the github project at https://github.com/caskdata/coopr/releases.
 
-Release Theme: 
------------------------------
-
 Release Highlights
 ------------------
-  * 1
-  * 2
+  * Docker support, ability to deploy Docker services and manage Docker containers
+  * Provider plugin for Digital Ocean added
+  * New UI, rewritten to be more modular and extensible
+  * CLI tool, allows interaction via simple commands
+  * Split Server APIs into internal and external, served on different ports
+  * Node usage metrics, provides detailed metrics by tenant, user, or template
+  * Significant bug and stability fixes to provisioner and server
+  * Initial support for versioned templates
+  * Initial SSL support for UI and server
+
+  * New Services: Impala, Node.js modules, PHP modules, Docker, Nginx 
+  * New Cluster Templates: Docker, MEAN, MongoDB
+  * Coopr cookbook updates (coopr_*)
 
 Change List
 -----------
-  * 1 (Issues: #xx #xx)
+
+New Features
+^^^^^^^^^^^^
+
+  * `COOPR-195 <https://issues.cask.co/browse/COOPR-195>`_ - Entity versioning
+  * `COOPR-199 <https://issues.cask.co/browse/COOPR-199>`_ - Ability to suspend jobs
+  * `COOPR-418 <https://issues.cask.co/browse/COOPR-418>`_ - Add more detailed usage statistics
+  * `COOPR-493 <https://issues.cask.co/browse/COOPR-493>`_ - Add a command line interface
+  * `COOPR-496 <https://issues.cask.co/browse/COOPR-496>`_ - Add some way to expose links for a cluster
```

Comment:
That's the JIRA's title... these are unedited. Should I still edit it? My gut says "leave it"